docs/tutorials/edc-cut-bell-pair-benchmarking.ipynb

```diff
@@ -0,0 +1,715 @@
+    "circuits[-1].draw(\"mpl\", fold=-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97eb3019",
+   "metadata": {},
+   "source": [
+    "## Step 2: Optimize problem for quantum hardware execution\n",
+    "\n",
+    "\n",
+    "Before executing our circuits on real hardware, we need to transpile them to match the device’s physical constraints. Transpilation will map the abstract circuit onto the physical qubits and gate set of the chosen device. Since we have already chosen specific physical qubits for each chain (by providing an `initial_layout` to the circuit generator), we use transpiler `optimization_level=0` with that fixed layout. This tells Qiskit not to reassign qubits or perform any heavy optimizations that could alter the circuit structure. We want to keep the sequence of operations (especially the conditional gates) exactly as specified.\n",
+    "\n",
+    "One important preparation is to ensure the transpiler knows about the dynamic circuit instructions. We explicitly add the `IfElseOp` (if/else construct) to the backend’s instruction set (`backend.target`) before transpiling. This lets the transpiler and scheduler know that our hardware can handle the conditional operations natively."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b36f5802",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isa_circuits = []\n",
+    "for ind, init_layout in enumerate(initial_layouts):\n",
+    "    pm = generate_preset_pass_manager(\n",
+    "        optimization_level=0, backend=backend, initial_layout=init_layout\n",
+    "    )\n",
+    "    isa_circ = pm.run(circuits[ind * 2 : ind * 2 + 2])\n",
+    "    isa_circuits.extend(isa_circ)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "id": "3ad620f7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Image src=\"/docs/images/tutorials/edc-cut-bell-pair-benchmarking/extracted-outputs/3ad620f7-0.avif\" alt=\"Output of the previous code cell\" />"
+      ]
+     },
+     "execution_count": 65,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "isa_circuits[1].draw(\"mpl\", fold=-1, idle_wires=False)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c5edec73",
+   "metadata": {},
+   "source": [
+    "## Step 3: Execute using Qiskit primitives\n",
+    "\n",
+    "\n",
+    "We can now run the experiment on the quantum device. We use Qiskit Runtime and its Sampler primitive to execute the batch of circuits efficiently."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f1a5ad8c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sampler = Sampler(mode=backend)\n",
+    "sampler.options.experimental = {\n",
+    "    \"execution_path\": \"gen3-experimental\",\n",
+    "    \"require_isa_circuits\": False,\n",
```

Comment:
`require_is_circuits` is not needed, and it's not a flag we normally advertise publicly.